src/Contracts/Markets/OpdexStandardMarket.cs

```diff
@@ -133,73 +134,34 @@ public override Address CreatePool(Address token)
         
         return pool;
     }
-    
-    /// <inheritdoc />
-    public override UInt256[] AddLiquidity(Address token, UInt256 amountSrcDesired, ulong amountCrsMin, UInt256 amountSrcMin, Address to, ulong deadline)
-    { 
-        EnsureAuthorizationFor(Message.Sender, Permissions.Provide);
-        
-        return AddLiquidityExecute(token, amountSrcDesired, amountCrsMin, amountSrcMin, to, deadline);
-    }
-    
+
     /// <inheritdoc />
-    public override UInt256[] RemoveLiquidity(Address token, UInt256 liquidity, ulong amountCrsMin, UInt256 amountSrcMin, Address to, ulong deadline)
+    public void CollectMarketFees(Address token, UInt256 amount)
     {
-        EnsureAuthorizationFor(Message.Sender, Permissions.Provide);
+        if (!MarketFeeEnabled || amount == 0) return;
```

Comment:
```suggestion
        if (amount == 0 || !MarketFeeEnabled) return;
```
Really minor an insignificant, could save the tiniest amount of gas